src/App/App.tsx

```diff
@@ -47,15 +47,25 @@ export default function App() {
               <Route path={`${appRoutes.donate}/:id`} element={<Donate />} />
               <Route path={appRoutes.leaderboard} element={<Leaderboard />} />
               <Route
+                index={IS_AST}
                 path={`${appRoutes.register}/*`}
                 element={IS_AST ? <Launchpad /> : <Registration />}
               />
               <Route path={`${appRoutes.gift}/*`} element={<Gift />} />
-              <Route index element={<Marketplace />} />
+              <Route
```

Comment:
Failing ["Lint and test the codebase" job](https://github.com/AngelProtocolFinance/angelprotocol-web-app/actions/runs/4674841625/jobs/8281870623?pr=2056#logs), there's probably an infinite loop in the `App` component-level router

Reply:
Resolved